scripts/deploy.js

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env node
+"use strict"
+
+var path = require('path')
+var sh = require('shelljs')
+var config = require('yargs')
+  .option('s', {
```

Comment:
I'm wondering if we need all these different options - we could probably simplify it a lot by cutting down the options we don't need/can always use the default.

Reply:
Keeping these, as per I want to. :D